### ISSUE TITLE
Data structure missing ?

### ISSUE THREAD

**Issue by pyladune** [rizafahmi/elixirjobs#30]
I may missed something but I have my rethinkDB working. However I do not see our to create the default tables to insert dev data. My rethink DB is empty. Should I create a database ? name ? 
where is the structure definition ? i do not see any migration in the code.

Thanks 

Seb

**Comment by rizafahmi** [comment]
Yes, sorry about that. I don't have that yet. Basicly you just need two tables: jobs and devs.

**Comment by optikfluffel** [comment]
I've added `r.tableCreate("jobs");` and `r.tableCreate("devs");` to `README.md` in #36.

@rizafahmi also there was/is `r.table("users")` in `README.md` so I guessed that this would be needed as well and also added `r.tableCreate("users");`
